server/keyshare/keyshareserver/server.go

```diff
@@ -288,22 +246,22 @@ func (s *Server) handleResponse(w http.ResponseWriter, r *http.Request) {
 	server.WriteString(w, proofResponse)
 }
 
-func (s *Server) doGenerateResponses(user KeyshareUser, authorization string, challenge *big.Int, commitID uint64, keyID irma.PublicKeyIdentifier) (string, error) {
+func (s *Server) generateResponses(user *User, authorization string, challenge *big.Int, commitID uint64, keyID irma.PublicKeyIdentifier) (string, error) {
```

Comment:
Is this function really necessary? It's called `generateResponses` but actually generates one response via the core function. So maybe integrate it where it is used and rather extract the database calls from the `handleRreponse` method?

Reply:
Instead the fetching of the session from `s.store` is now moved to this method, as discussed elsewhere.